src/ui/ResultList/ResultListConstants.ts

```diff
@@ -0,0 +1 @@
+export const resultListId = 'ResultList';
```

Comment:
Is creating a file here for 1 variable here really necessary when you already import stuff from ResultList into QuerySummaryUtils? I don't see a real problem with it being inside the ResultList file

Reply:
If I move this variable into ResultList and export it from there, I get have the circular dependency error. If I import just an interface, it is okay. For that reason, I put this variable in it's own file. That said, hard-coding the string could be another option. @olamothe ?

Reply:
Decided to hard-code and remove this one-line file for now.